packages/core/src/components/meter/meter.scss

```
@@ -0,0 +1,79 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
+
+  &.has-center-title {
+    justify-content: center;
+  }
+
+  &.has-gap-top {
+    margin-bottom: 0;
+    margin-top: var(--spacing-xsmall);
+  }
+
+  @media (max-width: 768px) {
```

block: we need to use sugar syntax here

```suggestion
  @include below(small) {
```

---

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/8ff4b7966701aa81d7bd386a71d41ff8c9103e58